src/features/surveys/components/SurveySubmissionsList.tsx

```diff
@@ -191,7 +203,7 @@ const SurveySubmissionsList = ({
         cell={row.respondent}
         onUpdate={updateCellValue}
         removePersonLabel={messages.submissions.unlink()}
-        suggestedPeople={suggestedPeople}
+        suggestedPeople={respondents}
         suggestedPeopleLabel={messages.submissions.suggestedPeople()}
```

Comment:
The list of suggested people (respondents) is unaffected by the search string, is there a specific reason for this? I found it a bit clunky to search and not have the suggested people be filtered by my search string.

Reply:
The filtering of the suggested people is happening in the `ZUIPersonEditGridCell `and I fix it to make it work :)